.github/workflows/refresh-lists.yml

```diff
@@ -44,8 +44,6 @@ jobs:
           echo -n "${{ steps.snap-matrix.outputs.matrix }}" |
             xargs -d' ' -n3 -P`nproc` \
               bash -xc "unsquashfs -cat \$0_\$2.snap snap/\$2.list \
-                | sed 's/\(.so.[0-9]\+\)\([0-9\.]\+\)\?$/\1*/' \
-                | sort -u \
```

Comment:
@sergio-costas FYI I dropped these as canonical/mesa-2404#50 does this on the snap side.

Reply:
This suggestion can be commited after #50 is merged.